lua/wire/wireshared.lua

```diff
@@ -999,3 +999,71 @@ function nicenumber.nicetime( n )
 		return "0s"
 	end
 end
+
+------------------------------------------------------
+-- Loads a font into the cache if it doesn't exist
+-- Returns true if font exists, else returns false
+------------------------------------------------------
+local _fontcache = {}
+local _defaultFontSize = 18
+
+function WireLib.LoadFont( name, size )
+	if not name then return nil end
+	
+	local fontSize = size or _defaultFontSize
+	local fontKey = name .. "_" .. fontSize
+	
+	-- Shortcircuit if font has already loaded
+	if _fontcache[fontKey] ~= nil then
+		return fontKey
+	end
+	
```

Comment:
`ReloadFont` was intended as a way of forcing the font to reload and bypassing the cache (i.e. if the cached font is a cache of an outdated font). Feel free to make one a wrapper of the other or remove `ReloadFont` if you feel this isn't necessary.